hi-ml-histopathology/testhisto/testhisto/mocks/tiles_datamodule.py

```diff
@@ -0,0 +1,165 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+import os
+import numpy as np
+import pandas as pd
+
+from typing import Any, Tuple
+import torch
+from torchvision.utils import save_image
+
+from health_ml.utils.split_dataset import DatasetSplits
+from histopathology.datamodules.base_module import TilesDataModule
+from histopathology.datasets.base_dataset import TilesDataset
+from testhisto.mocks.base_datamodule import MockHistoDataGenerator
+
+
+class MockTilesDataset(TilesDataset):
```

Comment:
should this be `MockPandaTilesDataset` given it has some settings specific to PANDA?